From 512fe6865738286c94f87da64ca6b475bc3e2f38 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Tue, 7 Jul 2015 10:13:14 -0400
Subject: [PATCH 04/90] Fix bzr's ignore artificats in .gitignore

We lost a man page because of incompatibilities between bzr's ignore
system and .gitignore, so solve that slightly better.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 .gitignore | 265 ++++++++++++++++++++++++++++---------------------------------
 1 file changed, 120 insertions(+), 145 deletions(-)

diff --git a/.gitignore b/.gitignore
index 18ab8e8..06c9f3a 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,93 +1,154 @@
-00_header
-10_*
-20_linux_xen
-30_os-prober
-40_custom
-41_custom
-*.1
-*.8
+!util/*.[[:digit:]]
+!util/grub.d/[[:digit:]][[:digit:]]_*.in
+*.[[:digit:]]
+*.a
+*.exe
+*.exec
+*.image
+*.img
+*.log
+*.lst
+*.marker
+*.mod
+*.o
+*.pf2
+*.pp
+*.trs
+.deps
+.deps-core
+.deps-util
+.dirstamp
+DISTLIST
+GPATH
+GRTAGS
+GSYMS
+GTAGS
+Makefile
+Makefile.in
+Makefile.util.am
+Makefile.utilgcry.def
+[[:digit:]][[:digit:]]_*
 aclocal.m4
 ahci_test
 ascii.bitmaps
 ascii.h
 autom4te.cache
+build-aux/test-driver
 build-grub-gen-asciih
 build-grub-gen-widthspec
 build-grub-mkfont
 cdboot_test
 cmp_test
+compile
+config-util.h
+config-util.h.in
 config.cache
 config.guess
 config.h
-config-util.h
-config-util.h.in
 config.log
 config.status
 config.sub
 configure
+contrib
 core_compress_test
-DISTLIST
+depcomp
 docs/*.info
+docs/stamp-1
 docs/stamp-vti
+docs/version-dev.texi
 docs/version.texi
 ehci_test
 example_grub_script_test
 example_scripted_test
 example_unit_test
-*.exec
-*.exec.exe
 fddboot_test
+garbage-gen
 genkernsyms.sh
 gensymlist.sh
 gentrigtables
-gentrigtables.exe
 gettext_strings_test
 grub-bin2h
-/grub-bios-setup
-/grub-bios-setup.exe
+grub-bios-setup
+grub-core/*.module
+grub-core/*.pp
+grub-core/Makefile.core.am
+grub-core/Makefile.gcry.def
+grub-core/bootinfo.txt
+grub-core/contrib
+grub-core/gdb_grub
+grub-core/genmod.sh
+grub-core/gensyminfo.sh
+grub-core/gmodule.pl
+grub-core/gnulib/alloca.h
+grub-core/gnulib/arg-nonnull.h
+grub-core/gnulib/c++defs.h
+grub-core/gnulib/charset.alias
+grub-core/gnulib/configmake.h
+grub-core/gnulib/float.h
+grub-core/gnulib/getopt.h
+grub-core/gnulib/langinfo.h
+grub-core/gnulib/locale.h
+grub-core/gnulib/ref-add.sed
+grub-core/gnulib/ref-del.sed
+grub-core/gnulib/stdio.h
+grub-core/gnulib/stdlib.h
+grub-core/gnulib/string.h
+grub-core/gnulib/strings.h
+grub-core/gnulib/sys
+grub-core/gnulib/unistd.h
+grub-core/gnulib/unitypes.h
+grub-core/gnulib/uniwidth.h
+grub-core/gnulib/warn-on-use.h
+grub-core/gnulib/wchar.h
+grub-core/gnulib/wctype.h
+grub-core/grub.chrp
+grub-core/kernel.img.bin
+grub-core/lib/libgcrypt-grub
+grub-core/modinfo.sh
+grub-core/rs_decoder.h
+grub-editenv
+grub-emu
+grub-emu-lite
+grub-file
+grub-fs-tester
+grub-fstest
+grub-glue-efi
+grub-install
+grub-kbdcomp
+grub-macbless
+grub-macho2img
+grub-menulst2cfg
+grub-mk*
+grub-mount
+grub-ofpathname
+grub-probe
+grub-reboot
+grub-render-label
+grub-script-check
+grub-set-default
+grub-shell
+grub-shell-tester
+grub-sparc64-setup
+grub-syslinux2cfg
 grub_cmd_date
 grub_cmd_echo
 grub_cmd_regexp
 grub_cmd_set_date
 grub_cmd_sleep
-/grub-editenv
-/grub-editenv.exe
-grub-emu
-grub-emu-lite
-grub-emu.exe
-grub-emu-lite.exe
 grub_emu_init.c
 grub_emu_init.h
-/grub-file
-/grub-file.exe
-grub-fstest
-grub-fstest.exe
 grub_fstest_init.c
 grub_fstest_init.h
 grub_func_test
-grub-install
-grub-install.exe
-grub-kbdcomp
-/grub-macbless
-/grub-macbless.exe
-grub-macho2img
-/grub-menulst2cfg
-/grub-menulst2cfg.exe
-/grub-mk*
-grub-mount
-/grub-ofpathname
-/grub-ofpathname.exe
-grub-core/build-grub-pe2elf.exe
-/grub-probe
-/grub-probe.exe
 grub_probe_init.c
 grub_probe_init.h
-/grub-reboot
+grub_script.tab.c
+grub_script.tab.h
+grub_script.yy.c
+grub_script.yy.h
 grub_script_blanklines
 grub_script_blockarg
 grub_script_break
-grub-script-check
-grub-script-check.exe
 grub_script_check_init.c
 grub_script_check_init.h
 grub_script_comments
@@ -113,136 +174,50 @@ grub_script_strcmp
 grub_script_test
 grub_script_vars1
 grub_script_while1
-grub_script.tab.c
-grub_script.tab.h
-grub_script.yy.c
-grub_script.yy.h
-grub-set-default
 grub_setup_init.c
 grub_setup_init.h
-grub-shell
-grub-shell-tester
-grub-sparc64-setup
-grub-sparc64-setup.exe
-/grub-syslinux2cfg
-/grub-syslinux2cfg.exe
 gzcompress_test
 hddboot_test
 help_test
-*.img
-*.image
-*.image.exe
 include/grub/cpu
+include/grub/gcrypt/g10lib.h
+include/grub/gcrypt/gcrypt.h
 include/grub/machine
 install-sh
 lib/libgcrypt-grub
 libgrub_a_init.c
-*.log
-*.lst
 lzocompress_test
-*.marker
-Makefile
-*.mod
-mod-*.c
+mdate-sh
 missing
+mod-*.c
 netboot_test
-*.o
-*.a
 ohci_test
 partmap_test
 pata_test
-*.pf2
-*.pp
+po/*.gmo
 po/*.mo
-po/grub.pot
+po/*.po
+po/LINGUAS
 po/POTFILES
+po/POTFILES-shell.in
+po/POTFILES.in
+po/grub.pot
+po/remove-potcdate.sed
 po/stamp-po
 printf_test
 priority_queue_unit_test
 pseries_test
 stamp-h
-stamp-h1
 stamp-h.in
+stamp-h1
 symlist.c
 symlist.h
+texinfo.tex
 trigtables.c
-*.trs
 uhci_test
-update-grub_lib
 unidata.c
-xzcompress_test
-Makefile.in
-GPATH
-GRTAGS
-GSYMS
-GTAGS
-compile
-depcomp
-mdate-sh
-texinfo.tex
-grub-core/lib/libgcrypt-grub
-.deps
-.deps-util
-.deps-core
-.dirstamp
-Makefile.util.am
-contrib
-grub-core/bootinfo.txt
-grub-core/Makefile.core.am
-grub-core/Makefile.gcry.def
-grub-core/contrib
-grub-core/gdb_grub
-grub-core/genmod.sh
-grub-core/gensyminfo.sh
-grub-core/gmodule.pl
-grub-core/grub.chrp
-grub-core/modinfo.sh
-grub-core/*.module
-grub-core/*.module.exe
-grub-core/*.pp
-grub-core/kernel.img.bin
+update-grub_lib
 util/bash-completion.d/grub
-grub-core/gnulib/alloca.h
-grub-core/gnulib/arg-nonnull.h
-grub-core/gnulib/c++defs.h
-grub-core/gnulib/charset.alias
-grub-core/gnulib/configmake.h
-grub-core/gnulib/float.h
-grub-core/gnulib/getopt.h
-grub-core/gnulib/langinfo.h
-grub-core/gnulib/ref-add.sed
-grub-core/gnulib/ref-del.sed
-grub-core/gnulib/stdio.h
-grub-core/gnulib/stdlib.h
-grub-core/gnulib/string.h
-grub-core/gnulib/strings.h
-grub-core/gnulib/sys
-grub-core/gnulib/unistd.h
-grub-core/gnulib/warn-on-use.h
-grub-core/gnulib/wchar.h
-grub-core/gnulib/wctype.h
-grub-core/rs_decoder.h
 widthspec.bin
 widthspec.h
-docs/stamp-1
-docs/version-dev.texi
-Makefile.utilgcry.def
-po/*.po
-po/*.gmo
-po/LINGUAS
-po/remove-potcdate.sed
-include/grub/gcrypt/gcrypt.h
-include/grub/gcrypt/g10lib.h
-po/POTFILES.in
-po/POTFILES-shell.in
-/grub-glue-efi
-/grub-render-label
-/grub-glue-efi.exe
-/grub-render-label.exe
-grub-core/gnulib/locale.h
-grub-core/gnulib/unitypes.h
-grub-core/gnulib/uniwidth.h
-build-aux/test-driver
-/garbage-gen
-/garbage-gen.exe
-/grub-fs-tester
+xzcompress_test
-- 
2.9.3

